eng/performance/scenarios.yml

```diff
@@ -102,6 +102,18 @@ jobs:
             displayName: Build SOD tool
             env:
               PERFLAB_TARGET_FRAMEWORKS: $(PERFLAB_Framework)
+          - ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
+            - powershell: |
+                mkdir $(CorrelationStaging)PDN
+                curl "https://pvscmdupload.blob.core.windows.net/assets/PDN3.zip$(PerfCommandUploadTokenNonEscaped)" -OutFile $(CorrelationStaging)PDN\PDN3.zip
```

Comment:
Is there any documentation explaining how to update this zip file for new PDN versions? Also is this a custom build of PDN that we have put in extra instrumentation or is it just the version straight from the PDN repo?

Reply:
Yeah, make sure to add a `README.md` or something that captures various update history to the zip file.

Reply:
I am just going to do this in a comment of the yml file itself